### DISCUSSION TITLE
Has development ended?

### DISCUSSION THREAD

**Discussion by ghost** [bleachbit/bleachbit#1459] [General]
4.4.2 was the last release over a year ago.

Is the project still active?

**Comment by az0** [comment]
>Has development ended? 

No

>Is the project still active?

Yes

>4.4.2 was the last release over a year ago.

There are new [development releases](https://ci.bleachbit.org/) from this week, plus here is a [change history ](https://github.com/bleachbit/bleachbit/commits/master) of the master branch.
